config.js

```diff
@@ -37,6 +37,30 @@ module.exports = {
     src: './src/assets/toolkit/images/*.*',
     dest: './dist/assets/toolkit/images',
   },
+  angularCore: {
+    src: './src/angular/src/app/spark-core-angular/dist/*.*',
+    dest: './packages/spark-core-angular',
+  },
+  angularExtrasCard: {
+    src:
+      './src/angular/src/app/spark-extras-angular/components/spark-extras-angular-card/dist/*.*',
```

Comment:
These paths are wrong. Did you accept the changes from my file or yours? Mine had the correct ones.

Reply:
yeah i think i goofed the merge a little, but these are coming out anyway right? since we're going to point lerna at the dist directories anyway

Reply:
Yeh, if we don't need them, then doesn't matter!